### DISCUSSION TITLE
Draw a deformation mechanism map

### DISCUSSION THREAD

**Discussion by Zhanlf0226** [mtex-toolbox/mtex#1283] [Ask Anything]
Dear all,

I want to draw a deformation mechanism map like this:
![3](https://user-images.githubusercontent.com/87469624/142173187-6d6b9f7c-3154-4826-9d01-f967a4775d38.png)
but I got this result using origin
![2](https://user-images.githubusercontent.com/87469624/142172537-45e66a38-0d16-48f1-8536-bfe873b60611.png)
The curves are not connected and smooth.
I want to know how to draw it in matlab.

Thanks,
LF

**Comment by lfgmorales** [comment]
Hi

This is not available on MTEX, but Jessica Warren made her nice matlab codes for deformation mechanism maps available here

https://www.jessicamwarren.com/research/research-rheology/

Cheers

Luiz


Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 17 Nov 2021, at 10:25, Zhanlf0226 ***@***.***> wrote:

﻿

Dear all,

I want to draw a deformation mechanism map like this:
[3]<https://user-images.githubusercontent.com/87469624/142173187-6d6b9f7c-3154-4826-9d01-f967a4775d38.png>
but I got this result using origin
[2]<https://user-images.githubusercontent.com/87469624/142172537-45e66a38-0d16-48f1-8536-bfe873b60611.png>
The curves are not connected and smooth.
I want to know how to draw it in matlab.

Thanks,
LF

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1283>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN57KDUYDXIKNMQ74ATUMNYGRANCNFSM5IGNUFAA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1283", "url": "https://github.com/mtex-toolbox/mtex/discussions/1283", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by Zhanlf0226** [reply]
Thank you！
